openml/extensions/sklearn/extension.py

```diff
@@ -1133,17 +1200,29 @@ def _prediction_to_probabilities(
 
         try:
             # for measuring runtime. Only available since Python 3.3
-            if can_measure_runtime:
-                modelfit_starttime = time.process_time()
+            modelfit_start_cputime = None
+            modelfit_duration_cputime = None
+            modelpredict_start_cputime = None
+
+            modelfit_start_walltime = None
+            modelfit_duration_walltime = None
+            modelpredict_start_walltime = None
+            if can_measure_cputime:
+                modelfit_start_cputime = time.process_time()
+            if can_measure_wallclocktime:
+                modelfit_start_walltime = time.time()
```

Comment:
Why not define these as
```
modelfit_start_cputime = time.process_time() if can_measure_cputime else None
```
I also don't think you have to define `modelfit_duration_X` here, as they are only used within their guarded if-statements.

Reply:
Because PyCharm does not like that. obviously it does not error but having no warnings ensures a good overview